### ISSUE TITLE
SyntaxError

### ISSUE THREAD

**Issue by fklement** [zuhito/node-red-contrib-sendgrid#2]
Hello,
i can't send an email via using the sendgrid node.
I tried different things but i'm always getting the same error message:
![error](https://user-images.githubusercontent.com/7302097/39083526-864c00ca-4565-11e8-8abb-69801c06ff56.PNG)
I'm setting the `msg.payload` and `msg.topic` via a template node.
Both are containing only plain text.

**Comment by zuhitocom** [comment]
Hmm... In my environment, the node works correctly. Could you share your simplest flow as JSON data?

**Comment by fklement** [comment]
Sure. The complete JSON looks like this:
```
{
	"msg": {
		"payload": "The refrigerator temperature is -19 °C",
		"topic": "Fridge info"
	}
}
```
![error_2](https://user-images.githubusercontent.com/7302097/39084621-e3188180-4578-11e8-91fc-3d88f83551e6.PNG)

The Template for the Message node looks like this:
-  `Set property` => `msg.payload`
-  `Format` => `Mustache template`
-  `Template` => `The refrigerator temperature is {{payload}} °C`
- `Output as` => `Plain text`

The Template for the Subject node looks like this:
-  `Set property` => `msg.topic`
-  `Format` => `Plain text`
-  `Template` => `Fridge info`
- `Output as` => `Plain text`

**Comment by zuhito** [comment]
Thank you for your sharing. I tried it. It works correctly and I received the e-mail from sendgrid node.

![unnamed](https://user-images.githubusercontent.com/6851138/39086189-f8e081c0-45c8-11e8-8675-e09600b57300.png)

Please try to use my flow. :)
```
[{"id":"860e0efa.3b3ee","type":"sendgrid","z":"ff1c2313.11ba7","from":"","to":"","name":"Sendmail","x":380,"y":260,"wires":[]},{"id":"4c2d43bf.e0024c","type":"inject","z":"ff1c2313.11ba7","name":"","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"x":120,"y":140,"wires":[["538a091d.4f1958"]]},{"id":"538a091d.4f1958","type":"template","z":"ff1c2313.11ba7","name":"Message","field":"payload","fieldType":"msg","format":"handlebars","syntax":"mustache","template":"The refrigerator temperature is {{payload}} °C","output":"str","x":140,"y":200,"wires":[["387010c1.2b588"]]},{"id":"387010c1.2b588","type":"template","z":"ff1c2313.11ba7","name":"Subject","field":"topic","fieldType":"msg","format":"handlebars","syntax":"plain","template":"Fridge info","output":"str","x":280,"y":200,"wires":[["860e0efa.3b3ee"]]},{"id":"c8228178.14e4b","type":"comment","z":"ff1c2313.11ba7","name":"Input API Key, From, To ->","info":"","x":190,"y":260,"wires":[]}]
```

**Comment by fklement** [comment]
Thanks for your efford.
I tried your flow and reinstalled the sendgrid node but i'm getting the same error.
What else could i try?

**Comment by fklement** [comment]
I completely reset my raspberry pi and reinstalled node-red from scratch.
Still getting the same error!
Do i need to configure something additional?

**Comment by zuhitocom** [comment]
Could you tell me your envrionment (Node- RED version, Node.js version, OS, browser). To survey the problem, I'll try to use the node on the same environment.

**Comment by fklement** [comment]
Sure. 
Raspberry PI:
```
Node-RED version: v0.18.4    
Node.js  version: v4.8.2  
Linux 4.14.34-v7+ arm LE  
```
Browser:
```Google Chrome: Version 65.0.3325.181 (Offizieller Build) (64-Bit)```

**Comment by zuhito** [comment]
I encountered same problem when I tried to use Node.js v4.4.5. After I upgraded it to Node.js v8.1.1, the problem was solved. Therefore, the problem occurs from the reason of Node.js version. If you use Raspberian on Raspberry Pi, "update-nodejs-and-nodered" command will be useful to upgrade your Node.js.
See the details of "update-nodejs-and-nodered" command.
https://nodered.org/docs/hardware/raspberrypi
